src/igroupadmin.cpp

```diff
@@ -290,8 +290,9 @@ doCommand( char *cmdToken[] ) {
         return 0;
     }
 
+    char empty_string[] = "";
```

Comment:
Looks like this can be moved inside of the if-stmt.

Reply:
done